File: SmartWash/src/main/java/com/smartwash/vo/users/UserInfoVo.java
package com.smartwash.vo.users;

import com.smartwash.vo.schools.SchoolsVo;
import lombok.Data;

import java.math.BigDecimal;

@Data
public class UserInfoVo {
    private String phoneNumber;

    private String studentId;

    private String campusCard;

    private BigDecimal balance;

    private String avatar;

    private SchoolsVo schoolVo;
}

File: SmartWash-Android/app/src/main/java/com/smartwash/network/vo/user/UserInfoVo.kt
package com.smartwash.network.vo.user

import androidx.annotation.Keep
import com.smartwash.network.vo.school.SchoolVo

@Keep
data class UserInfoVo(
    val phoneNumber: String,
    val studentId: String,
    val campusCard: String,
    val balance: Float,
    val schoolVo: SchoolVo,
)
